internal/repositories/mongo/team/get.go

```diff
@@ -2,10 +2,26 @@ package team
 
 import (
 	"context"
+	"fmt"
+	"log"
 	"github.com/jairogloz/go-l/internal/domain"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
 )
 
 func (r Repository) Get(ctx context.Context, id string) (team domain.Team, err error) {
```

Comment:
Es buen momento para inciar con la buena práctica de agregar comentarios a las funciones 🤓 . Los comentarios deben empezar con el nombre de la función, seguido de una descripción de lo que la función hace. Ej.

// Get gets a player from the repository identified by the given id...

Reply:
Excelente, voy a comenzar a aplicarlo